src/CONST.js

```diff
@@ -672,6 +672,12 @@ const CONST = {
         GSD: 'gsd',
         DEFAULT: 'default',
     },
+    THEME: {
+        LIGHT: 'light',
+        DARK: 'dark',
+        SYSTEM: 'system',
+    },
+    DEFAULT_THEME: 'dark',
```

Comment:
Let's go ahead and move `DEFAULT_THEME` under the `THEME` key?

Reply:
I was thinking the same initially, but wouldn't that implicate, that there is a separate `DEFAULT` theme, with a different color palette? Like `dark` and `light`...

Any way is fine with me though